src/core/tsi/ssl_transport_security.h

```diff
@@ -360,6 +364,8 @@ struct tsi_ssl_server_handshaker_options {
   // will be unusable.
   bool send_client_ca_list;
 
+  std::shared_ptr<grpc_core::SpiffeBundleMap> spiffe_bundle_map;
```

Comment:
I suggest combining the `pem_client_root_certs` and `spiffe_bundle_map` fields using the same `std::shared_ptr<RootCertInfo>` type that I've suggested elsewhere.